compiler/fir/checkers/src/org/jetbrains/kotlin/fir/analysis/checkers/declaration/FirConflictsChecker.kt

```diff
@@ -322,6 +329,22 @@ object FirConflictsChecker : FirBasicDeclarationChecker() {
         }
     }
 
+    private fun checkBlock(body: FirBlock, inspector: DeclarationInspector) {
```

Comment:
The method doesn't check in the following case:

```kt
fun main(b: Boolean) {
    if (b) {
        val someVal2 = 2
        val someVal2 = "Two"
    }
}
```

Please consider more cases.